modules/test/playwright/helpers/SearchExperiencesApiHelper.ts

```diff
@@ -69,7 +54,13 @@ export class SearchExperiencesApiHelper {
 		description = '',
 		elementInstances = [],
 		title = `Blueprint${getRandomInt()}`,
-	}: createSXPBlueprintProps = {}): Promise<{id: number}> {
+	}: {
+		configuration?: Object;
+		defaultLanguageId?: string;
+		description?: string;
+		elementInstances?: Object[];
+		title?: string;
+	} = {}): Promise<{id: number}> {
```

Comment:
Do you need the `= {}` part?

Reply:
Also I noticed `Promise<{id: number}>, but this is returning the entire blueprint object not just the `id`.

Reply:
Ah I see, makes sense.

Reply:
Added in types for blueprint and element 👍  Should be a lot more organized now, thanks for catching!!